tf-psa-crypto/drivers/builtin/include/mbedtls/ecp.h

```diff
@@ -200,7 +200,8 @@ mbedtls_ecp_point;
  *
  * For Montgomery curves, we do not store \p A, but <code>(A + 2) / 4</code>,
  * which is the quantity used in the formulas. Additionally, \p nbits is
- * not the size of \p N but the required size for private keys.
+ * not the size of \p N but the position of the highest bit for private keys,
```

Comment:
For maximum clarity, it might be best to mirror the language in ecp_curves.c. For example,

```
the required msb for private keys, i.e. the private key bit-size minus 1.
```

Reply:
I don't find this clearer. In fact part of me wants to change “the required msb for private keys” to “the position of the most significant bit for private keys” (and another part doesn't want to bother changing the comment which is in a place where if you're digging, you should know what you're doing).

Reply:
Ironically, "the position of the most significant bit for private keys" was my first instinct.